README.md

```diff
@@ -241,6 +241,32 @@ installed modules. It is recommended to add `node-modules` to your
 If you are using `lein-npm`, follow their
 [instructions](https://github.com/RyanMcG/lein-npm).
 
+#### Measuring coverage with Istanbul
+
+It's possible to generate Istanbul coverage reports for JS files produced from CLJS.
+
+To make it work two things are required.
+
+Install your karma coverage plugin.
+
+    npm install karma-coverage --save-dev
+
+Add coverage seetings to your `project.clj`
+
+    :doo {:coverage {:packages [:my-app.module]
```

Comment:
Small change suggested: to match the way e.g. cljsbuild :main is configured, it'd be nice to have packages be symbols instead of keywords. Your code already supports it, but it could be shown here: `:packages [my-app.module]`.